src/shared/components/GeneSelectionBox/GeneSymbolValidator.tsx

```diff
@@ -37,7 +37,7 @@ export default class GeneSymbolValidator extends React.Component<IGeneSymbolVali
 					<span className={styles.errorMessage}>
 						{`Cannot validate gene symbols because of invalid OQL. ${
 							this.props.geneQueryErrorDisplayStatus === DisplayStatus.UNFOCUSED
-							? "Please click 'Submit' to see location of error."
```

Comment:
This is just a temp fix since the component is only used in study view. There is another ticket for a more detailed solution. https://github.com/cBioPortal/cbioportal/issues/5378